services/user-services.js

```diff
@@ -0,0 +1,8 @@
+
+const userDao = require('../models/user.dao');
+
+const postUsers = async() => {
+  return await userDao.postUsers();
+}
```

Comment:
- 회원가입과 관련된 서비스 단의 함수 이름을 조금 더 명확하게 작성해주면 좋을 것 같습니다.
새로운 유저 정보를 생성하는 API의 비즈니스 로직이기 때문에 postUsers보다는 아래와 같은 함수명이 명확해보입니다.

- 함수의 arguments로 유저가 회원가입 시 보낸 데이터를 포함해주어야 합니다.

- 추가로 비밀번호는 bcrypt 모듈을 통해 암호화를 할 수 있는 코드를 추가해주세요.
```suggestion
const createUser = async(lastName, firstName, email, password, skinType) => {
  // bcrypt 모듈을 통한 비밀번호 암호화 로직 추가
```